lib/charms/layer/apache_spark.py

```diff
@@ -233,7 +356,11 @@ def start(self):
         # stop services (if they're running) to pick up any config change
         self.stop()
         # always start the history server, start master/worker if we're standalone
-        utils.run_as('ubuntu', '{}/sbin/start-history-server.sh'.format(spark_home), 'hdfs:///user/ubuntu/directory')
+        history_server_event_logs = '/tmp/spark-events'
```

Comment:
As we said the events log dir is set to /spark-history/tmp to match what we have for mapred